lib/DueDatePicker/styles.scss

```diff
@@ -0,0 +1,349 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+$duedate-popover-width: 304px !default;
+$duedate-timepicker-width: 80px !default;
+$duedate-timepicker-height: 220px !default;
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+.duedate__popover.duedate__popover {
+  max-width: $duedate-popover-width;
+  width: $duedate-popover-width;
+  margin-top: $layout-spacing-base / -2;
+
+  .arrow {
+    display: none;
+  }
+
+  &.duedate__popover .popover-content {
+    padding: 0;
+  }
+}
+
+.duedate__container {
+  font-size: $typo-size-slight;
+  position: relative;
+}
+
+.duedate__button {
+  color: $color-text-light;
+
+  .button {
+    text-decoration: underline;
+    color: $color-text-light;
+  }
+
+  .icon {
+    margin-right: $layout-spacing-base / 4;
+    top: -2px;
```

Comment:
Is this to centre the icon? Is so we could use flexbox.

Reply:
@kyleharper if I add display: flex it seems to mess with the positioning of the popover. Not sure of how to get around it. I might leave it as is for now. And when we come to redo popovers we can redo it then maybe? what you think

Reply:
Sounds good to me 👍